### PR TITLE
feat: add limited support for reading ply files

### PR DESCRIPTION
**Pull request by jingpengw**

### CODE REVIEW

File: zmesh/mesh.py [hunk 1]
```diff
@@ -144,6 +144,25 @@ def to_obj(self):
...
+    header, _, data = plydata.partition(b'end_header\n')
+    lines = header.splitlines()
+    assert lines[0] == b'ply'
+    assert lines[1] == b'format binary_little_endian 1.0'
+    vertexct = int(lines[2].decode('utf-8').split()[-1])
+    trianglect = int(lines[6].decode('utf-8').split()[-1])
+
+    # interpreate the data
+    vertices = np.frombuffer(data, dtype=np.float32, count=3*vertexct, offset=0).reshape(vertexct, 3)
+    faces = np.frombuffer(data, dtype=np.uint32, offset=vertexct*3*4).reshape(trianglect, 4)[:, 1:]
```
Comment (william-silversmith) [inline zmesh/mesh.py]: Why is the reshape argument 4? Below it's 3.

File: .gitignore [hunk 2]
```diff
@@ -42,4 +42,7 @@ __pycache__
 .pytest_cache
 
 # VS Code
-.vscode
\ No newline at end of file
+.vscode
+
+# mesh files
+*.ply
```
Comment (william-silversmith) [inline .gitignore]: Might be good to add .obj here too.

Comment (jingpengw) [reply .gitignore]: there is already one above.

File: zmesh/mesh.py (hunk 1 above)
Comment (jingpengw) [reply zmesh/mesh.py]: The binary data always have a '3' in the first column. Is that a bug of `to_ply` function?

Comment (jingpengw) [reply zmesh/mesh.py]: you have inserted 3s here: 
https://github.com/seung-lab/zmesh/blob/master/zmesh/mesh.py#L172

**Pull request merged**